src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/AbstractCreateSDORespondentNotificationSender.java

```diff
@@ -28,11 +28,15 @@ public abstract class AbstractCreateSDORespondentNotificationSender implements N
     void notifyRespondentPartySDOTriggered(CaseData caseData) {
         String email = getRecipientEmail(caseData);
         if (StringUtils.isNotBlank(email)) {
+            String sdoTemplate = notificationsProperties.getSdoOrdered();
```

Comment:
I would extract this into a private method getSDOTemplate()

Reply:
created `getSDOTemplate` method